lldb/source/Plugins/SymbolFile/DWARF/DWARFDebugInfoEntry.cpp

```diff
@@ -614,7 +614,7 @@ DWARFDebugInfoEntry::GetAbbreviationDeclarationPtr(const DWARFUnit *cu) const {
 }
 
 bool DWARFDebugInfoEntry::IsGlobalOrStaticScopeVariable() const {
-  if (Tag() != DW_TAG_variable)
+  if (Tag() != DW_TAG_variable && Tag() != DW_TAG_member)
```

Comment:
This patch is about supporting `DW_TAG_member`s that are static variables right? Wouldn't those be children of `DW_TAG_subprogram`s and thus fail this check anyway?